### PR TITLE
Some fixes for errant data, modifications to best catch paths.

### PR DESCRIPTION
**Pull request by nukes327**
Some data fixes that have been sitting around waiting for enough to make a reasonable PR.

### CODE REVIEW

**Comment by icykoneko** [comment]
Will get to this soon.

File: private/fishData.yaml [hunk 1]
```diff
@@ -9998,10 +10000,10 @@
   - Desert Dessert Frog
   predators:
     Sand Egg: 10
-  intuitionLength: 350
+  intuitionLength: 360
```
Comment (icykoneko) [inline private/fishData.yaml:10003]: I think this was correct.  350s = 2 Eorzean hours.

**Review by icykoneko — CHANGES_REQUESTED** [review]

**Comment by icykoneko** [comment]
LOL, you ended up grabbing the KO region data in this as well (re #122).  I'll double check it on my end, but it looked complete.

**Comment by nukes327** [comment]
If I remember correctly, Cinder Surprise's intuition starts at 6m and drops right after to 5m, which would mean 360s.

Let me find a file really quick to be sure.

**Comment by icykoneko** [comment]
I dunno, TC doesn't appear to record it, or doesn't publicize it.  It just seemed to make more sense being 2 eorzean hours.

**Comment by nukes327** [comment]
There was a big debate in the fishcord discovery at one point about why it would be just longer than 2 hours yeah.

6 minute rollover is visible in [Fruity's video here:](https://youtu.be/2S1OHYIcZrk?t=140)
firefox_2022-05-30_18-59-49

**Comment by icykoneko** [comment]
Okay then.  Uhhh, that's the only issue I had with it.  Looks like you may need to rebase your branch though after my commit today.

**Review by icykoneko — APPROVED** [review]

**Comment by nukes327** [comment]
Think that should do it

**Pull request merged**